yarn.lock

```diff
@@ -7556,7 +7561,22 @@ media-typer@0.3.0:
   resolved "https://registry.yarnpkg.com/media-typer/-/media-typer-0.3.0.tgz#8710d7af0aa626f8fffa1ce00168545263255748"
   integrity sha1-hxDXrwqmJvj/+hzgAWhUUmMlV0g=
 
-memcached@2.2.2, memcached@^2.2.2:
+memcache-client@^1.0.5:
+  version "1.0.5"
+  resolved "https://registry.yarnpkg.com/memcache-client/-/memcache-client-1.0.5.tgz#7a2eb1f1924b8a87ef7f80a25f42949da4d1eff7"
+  integrity sha512-hUADvh+Cju3eNZ5AWGfNhdI/Y4251pc8Hxq+6RryiVNzMeEnXKQ2z+gyBBAPY4SZnC1nOtp2MJUSsnCWB9t6lg==
+  dependencies:
+    lodash.defaults "4.2.0"
+    memcache-parser "^1.0.0"
+    optional-require "1.1.8"
+    tslib "2.1.0"
+
+memcache-parser@^1.0.0:
+  version "1.0.1"
+  resolved "https://registry.yarnpkg.com/memcache-parser/-/memcache-parser-1.0.1.tgz#66b8434e1bccd79a6da4bfe5fec41e5aa6f585b8"
+  integrity sha512-F5TjjHKUNMCWTVfiJD+7M9ecNbBdnK6xifYZ1WL8PoQ/dP9sB37VQV8hPtyuUqJJFj76qY70Ezj9saQ0gw2FJg==
+
+memcached@^2.2.2:
```

Comment:
Looks like the rate limit middleware still brings in `memcached`...maybe we should just clean that middleware up?

Reply:
I dug through slack for some history and couldn't figure out the state of this, or if it was _ever_ enabled. Ideally, `rate-limit-memcached` should be removed (and re-added if/when needed). I also wonder if the presence of both libraries could interfere with Datadog's tracing.

Which reminds me: we must confirm that datadog automatically instruments this alternate library.

Reply:
If we do remove `rate-limit-memcached`, we can also remove the mock `increment` implementation in `cache.ts` above.